source/hardware/controllers/ion_discover_dj_pro.rst

```diff
@@ -0,0 +1,94 @@
+
+.. note:: Only controls are documented that are known to be working.
+
+Deck section
+~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro_deck.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro (deck section)
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro (deck section)
+
+===  ======================================================   ============================================================================================
+No.  Control                                                  Function
+===  ======================================================   ============================================================================================
+1    :hwlabel:`JOG WHEEL`                                     Scratches the track
+2    :hwlabel:`PLAY/PAUSE` button                             Plays/pauses a track in the respective Deck
+3    :hwlabel:`SYNC` button                                   Sync the tempo to other playing track
+4    :hwlabel:`SCRATCH` button                                Toggle the scratch mode
+5    :hwlabel:`TEMPO` slider                                  Adjust the track playing speed
+6    :hwlabel:`PITCH BEND` buttons -/+                        Lower (-) or Raise (+) the pitch temporarily
+7    :hwlabel:`REVERSE` button                                Hold for reverse play
+8    :hwlabel:`BASS`/:hwlabel:`MID`/:hwlabel:`TREBLE` knobs   Adjust the low/mid/high-frequency regions of the song
+9    :hwlabel:`MUTE` buttons                                  Mute low/mid/high-frequency regions
+===  ======================================================   ============================================================================================
+
+Mixer section
+~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro_mixer.svg
+   :align: center
+   :width: 50%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro (deck section)
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro (mixer section)
+
+===  ======================================================   ============================================================================================
+No.  Control                                                  Function
+===  ======================================================   ============================================================================================
+1    Channel faders                                           Adjust the output level for each channel
+2    Cross fader                                              Fades between left and right deck.
+3    :hwlabel:`A CUE` / :hwlabel:`B CUE` buttons              Sends the selected channel's audio to the headphone output
+===  ======================================================   ============================================================================================
+
+Known Issues
+------------
+
+Controls not yet working (or tested):
+
+- Library browsing
```

Comment:
I think mapping the browse knob and the deck load buttons is a critical feature and shouldn't be hard to map). Could you take care of that?

Reply:
Welll... indeed the browse button + deck load buttons seems to work. So I can document that. I'll update the documentation on that one.

Reply:
Done